apps/etterlatte-saksbehandling-ui/client/src/components/person/SakOversikt.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
+import { Behandlingsliste } from './behandlingsliste'
+import styled from 'styled-components'
+import { IBehandlingListe } from './typer'
+import { ManueltOpphoerModal } from './ManueltOpphoerModal'
+import { hentBehandlingerForPerson } from '~shared/api/behandling'
+import { GridContainer } from '~shared/styled'
+import Spinner from '~shared/Spinner'
+import RelevanteHendelser from '~components/person/uhaandtereHendelser/RelevanteHendelser'
+import { isFailure, isPending, isSuccess, useApiCall } from '~shared/hooks/useApiCall'
+import { Alert, BodyShort, Heading, Link, Tag } from '@navikt/ds-react'
+import { ISak } from '~shared/types/sak'
+import { formaterSakstype } from '~utils/formattering'
+import { ExternalLinkIcon } from '@navikt/aksel-icons'
+
+export const SakOversikt = ({ fnr }: { fnr: string }) => {
+  const [sak, setSak] = useState<ISak>()
+  const [behandlingerStatus, hentBehandlinger] = useApiCall(hentBehandlingerForPerson)
+
+  useEffect(() => {
+    hentBehandlinger(fnr, (behandlinger: IBehandlingListe[]) => {
+      setSak(behandlinger[0].sak)
```

Comment:
Her kan du vel bruke `behandlinger.sak` slik at du slipper å ha denne som optional videre i koden

Reply:
Nei, ikke helt. Navn her er litt misvisende (gammel moro som burde ryddes). `BehandlingListe` er egentlig _saker_ med tilhørende behandlinger. Så jeg lar det være slik enn så lenge. Vi må på et tidspunkt lage støtte for at folk potensielt kan ha flere saker, men sannsynligheten for at det skjer er så liten at vi kan la det ligge en stund til (håper jeg).